test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectManagement/ServerHandlerTests.cs

```diff
@@ -47,5 +49,59 @@ public async Task GetServerProperties()
                 Assert.That(((ServerInfo)result.ViewInfo.ObjectInfo).IsPolyBaseInstalled, Is.Not.Null, $"Server isPolyBaseInstalled property should not be null");
             }
         }
+
+        /// <summary>
+        /// Test SetMemoryProperties for Sql Server
+        /// </summary>
+        [Test]
+        public async Task SetMemoryProperties()
+        {
+            var connectionResult = await LiveConnectionHelper.InitLiveConnectionInfoAsync("master", serverType: TestServerType.OnPrem);
+            using (SqlConnection sqlConn = ConnectionService.OpenSqlConnection(connectionResult.ConnectionInfo))
+            {
+                var server = new Server(new ServerConnection(sqlConn));
+                var serverHandler = new ServerHandler(ConnectionService.Instance);
+                var serverConfig = new ServerConfigService();
+
+                var requestParams = ObjectManagementTestUtils.GetInitializeViewRequestParams(connectionResult.ConnectionInfo.OwnerUri, "master", true, SqlObjectType.Server, "", "");
+                var result = (ServerInfo)(await serverHandler.InitializeObjectView(requestParams)).ViewInfo.ObjectInfo;
+                ServerInfo serverInfo = new ServerInfo()
+                {
+                    Name = result.Name,
+                    HardwareGeneration = result.HardwareGeneration,
+                    Language = result.Language,
+                    MemoryInMB = result.MemoryInMB,
+                    OperatingSystem = result.OperatingSystem,
+                    Platform = result.Platform,
+                    Processors = result.Processors,
+                    IsClustered = result.IsClustered,
+                    IsHadrEnabled = result.IsHadrEnabled,
+                    IsPolyBaseInstalled = result.IsPolyBaseInstalled,
+                    IsXTPSupported = result.IsXTPSupported,
+                    Product = result.Product,
+                    ReservedStorageSizeMB = result.ReservedStorageSizeMB,
+                    RootDirectory = result.RootDirectory,
+                    ServerCollation = result.ServerCollation,
+                    ServiceTier = result.ServiceTier,
+                    StorageSpaceUsageInMB = result.StorageSpaceUsageInMB,
+                    Version = result.Version,
+                    MinServerMemory = result.MinServerMemory,
+                    MaxServerMemory = result.MaxServerMemory
+                };
+
+                // Change memory settings
+                serverInfo.MinServerMemory = 10;
+                serverInfo.MaxServerMemory = 500;
+
+                Assert.AreNotEqual(result.MinServerMemory, serverInfo.MinServerMemory, "Server property should not be equal after update");
```

Comment:
We've started using the NUnit constraints for our newer C# tests. It helps make things a bit more readable if you want to use them with these tests. An example of the constraints: https://docs.nunit.org/articles/nunit/writing-tests/constraints/AndConstraint.html#examples-of-use

Reply:
I'll send out another pr tomorrow to add minimum and maximum values of the memory properties. I'll make sure to update the tests :)

Reply:
I just want to make sure this change gets in for the bug bash